plugins/dev9ghzdrk/Win32/ata.cpp

```diff
@@ -2957,7 +2957,7 @@ void ata_term()
 }
 #define LOG_IRS (sz!=2)
 template<int sz>
-u8 CALLBACK ata_read(u32 addr)
+extern "C" EXPORT u8 ata_read(u32 addr)
```

Comment:
I don't think EXPORT needs to be added to this. Only the DEV9* and PS2E* functions really need to be visible. The CALLBACK was probably unnecessary in the first place.

Also, this file isn't even compiled on Windows.

Reply:
Should we remove this entire file? What was it ever used for?

Reply:
Dunno. I've barely looked at the file. I wasn't involved with PCSX2 at that point.

Reply:
Looks like hard drive code adapted from QEMU. The plugin probably was trying to do hd emulation as well as networking at some point.

Reply:
https://github.com/jackun/USBqemu-wheel seems to have that abiliy (as well as Linux logitech headset capability I needed for SOCOM II online) . Would love to see that integrated at some point